### ISSUE TITLE
Fatal: The configuration file '_config.local.yml' could not be found.

### ISSUE THREAD

**Issue by likemusic** [magento/devdocs#1900]
## This issue is a:
[ ] Bug on the website
[ ] New topic request
[ ] Topic clarification request
[ ] New DevDocs feature request
[x] Other
 

## Description:
Got fatal error when run `rake preview`.
 

### Steps to reproduce
 
1. `rake preview`
 

## Expected result:
 Success build.

## Additional information:
```
Generating devdocs locally ... Configuration file: /root/devdocs/_config.yml
            Source: /root/devdocs
       Destination: /root/devdocs/_site
 Incremental build: enabled
      Generating...
jekyll 3.7.3 | Error:  No such file or directory @ rb_sysopen - /root/devdocs/guides/v2.2/architecture/arch-basics.md
enabled additional configuration parameters from _config.local.yml
Configuration file: _config.yml
             Fatal: The configuration file '_config.local.yml' could not be found.
Traceback (most recent call last):
        16: from bin/jekyll:16:in `<main>'
        15: from bin/jekyll:16:in `load'
        14: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/exe/jekyll:15:in `<top (required)>'
        13: from /usr/local/rvm/gems/ruby-head/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
        12: from /usr/local/rvm/gems/ruby-head/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
        11: from /usr/local/rvm/gems/ruby-head/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
        10: from /usr/local/rvm/gems/ruby-head/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
         9: from /usr/local/rvm/gems/ruby-head/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
         8: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
         7: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll/commands/serve.rb:89:in `start'
         6: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll/command.rb:43:in `configuration_from_options'
         5: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll.rb:110:in `configuration'
         4: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll/configuration.rb:202:in `read_config_files'
         3: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll/configuration.rb:202:in `each'
         2: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll/configuration.rb:204:in `block in read_config_files'
         1: from /usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll/configuration.rb:176:in `read_config_file'
/usr/local/rvm/gems/ruby-head/gems/jekyll-3.7.3/lib/jekyll/configuration.rb:188:in `rescue in read_config_file': The Configuration file '_config.local.yml' could not be found. (LoadError)
```

```
root@magentodevdocs:~/devdocs# ruby -v
ruby 2.6.0dev (2018-05-05 trunk 63341) [x86_64-linux]
```

```
root@magentodevdocs:~/devdocs# git log -1
commit 9af288e4f3999312ec80b37a48636cb3bca468da
Merge: 38b62fa cb93abc
Author: Kevin Harper <keharper@users.noreply.github.com>
Date:   Fri May 11 16:49:40 2018 -0500

    Merge pull request #1997 from magento/kh_pr1801

    delete unused files
```

```
root@magentodevdocs:~/devdocs# gem -v
2.7.6
```

```
root@magentodevdocs:~/devdocs# cat /etc/*-release
PRETTY_NAME="Debian GNU/Linux 8 (jessie)"
NAME="Debian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"
ID=debian
HOME_URL="http://www.debian.org/"
SUPPORT_URL="http://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

**Comment by likemusic** [comment]
After run `$ docker-compose up --force-recreate` in docker toolbox for window I didn't catch the error from previous comment and have access to http://192.168.99.100:4000/ with generated devdocs.

**Comment by likemusic** [comment]
Seems then when recreate top container docker uses cached intermediate containers. Therefore I didn't catch error.

Or maybe its another not related to previous bug.

```
$ docker-compose build --force-rm --no-cache
Building devdocs
Traceback (most recent call last):
  File "site-packages\docker\utils\utils.py", line 118, in create_archive
  File "tarfile.py", line 1977, in addfile
  File "tarfile.py", line 251, in copyfileobj
  File "tempfile.py", line 483, in func_wrapper
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "docker-compose", line 6, in <module>
  File "compose\cli\main.py", line 71, in main
  File "compose\cli\main.py", line 127, in perform_command
  File "compose\cli\main.py", line 280, in build
  File "compose\project.py", line 372, in build
  File "compose\service.py", line 998, in build
  File "site-packages\docker\api\build.py", line 150, in build
  File "site-packages\docker\utils\build.py", line 15, in tar
  File "site-packages\docker\utils\utils.py", line 121, in create_archive
OSError: Can not read file in context: D:\!Valera\!NOW\Magento 2\devdocs\_site\guides\v2.1\get-started\order-tutorial\order-prepare-checkout.html
[9904] Failed to execute script docker-compose
```

**Comment by likemusic** [comment]
Close this issue because seems that it fixed in #1904.
